docroot/modules/custom/mass_content/mass_content.deploy.php

```diff
@@ -317,3 +316,10 @@ function mass_content_deploy_header_media_images_followup(&$sandbox) {
     return t('All Header Media image caption fields migrated.');
   }
 }
+
+/**
+ * Migrate unpublished Header Media image wrapping fields.
+ */
+function mass_content_deploy_header_media_images_unpublished(&$sandbox) {
+  mass_content_deploy_header_media_images_followup($sandbox);
```

Comment:
@weitzman For this ticket, this deploy hook needs to do mostly the same thing as the previous deploy hook except it needs modify all field values regardless of the content's publication status. I am slightly hesitant to just rename and modify the old deploy hook since those names get stored in key_value, but is there a strong case either way between going with the approach I'm taking here vs renaming an old deploy hook?

Reply:
I think this approach is fine, except `mass_content_deploy_header_media_images_unpublished` now operates on both published and unpublished. If that’s desired then perhaps rename `mass_content_deploy_header_media_images_unpublished`.